templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -228,10 +228,42 @@
+      "text/plain": [
+       "<Figure size 720x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "TODO: add visualization"
+    "fovs = ['fov6', 'fov7']\n",
+    "for fov in fovs:\n",
```

Comment:
Let's try and remove some of these details from the user. From my comment about, the for loop should get integrated into the relabeling fucntion, so that it takes an xarray and relabels all of them. Then the plotting function can take the name of a single FOV within the xarray, and produce the visualization of just that FOV